docs/docs/11_deploy_to_render/06_run_everything_docker_compose/end/requirements.txt

```diff
@@ -0,0 +1,10 @@
+flask==2.3.2
```

Comment:
Should be flask version be pinned?

Maybe when adding `flask-smorest` `flask` can be removed since it is installed as a dependency?

Reply:
Aiming to pin dependencies, this could be removed now but I do want to pin everything (we've got something in Linear to do that work later on), so for now I think it doesn't matter 🤔